### ISSUE TITLE
Function plot_history in fuel efficiency, the regression example is not synchronized with the notebook

### ISSUE THREAD

**Issue by yiskylee** [tensorflow/tensorflow#25108]
<em>Please make sure that this is a documentation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:doc_template</em>

**System information**
- TensorFlow version: 1.12.0
- Doc Link: https://www.tensorflow.org/tutorials/keras/basic_regression

**Describe the documentation issue**
The function plot_history(history) is using pandas DataFrame "hist" from outside of the function scope, making it draw the same figure even when the EarlyStopping callbacks are used the second time. This error is corrected in the notebook (https://github.com/tensorflow/docs/blob/master/site/en/tutorials/keras/basic_regression.ipynb), but for some reason, it is not reflected on the website.

**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**
I will look into it later.

**Comment by yiskylee** [comment]
@lamberta thanks! https://www.tensorflow.org/tutorials/eager/custom_training_walkthrough has the same issue, so I think I would just leave it here:

plt.scatter(features['petal_length'],
            features['sepal_length'],
            c=labels,
            cmap='viridis')

should use "c=labels.numpy()", which is corrected in the colab notebook, but not in the tutorial page.

**Comment by yiskylee** [comment]
Looks great! Thanks.

**Comment by lamberta** [comment]
Thanks
